libp2p/protocols/secure/secio.nim

```diff
@@ -259,17 +260,16 @@ method write*(
     await sconn.stream.write(msg)
     sconn.activity = true
 
-proc newSecioConn(conn: Connection,
-                  hash: string,
-                  cipher: string,
-                  secrets: Secret,
-                  order: int,
-                  remotePubKey: PublicKey): SecioConn
-                  {.raises: [LPError].} =
```

Comment:
it doesn't actually raise, so use default `{.raises: [].}` from top level `{.push.}`.